docs/csharp/asynchronous-programming/snippets/generate-consume-asynchronous-streams/finished/Program.cs

```diff
@@ -139,9 +139,16 @@ await foreach (var issue in RunPagedQueryAsync(client, PagedIssueQuery, "docs")
         // Stop with 10 pages, because these are large repos:
         while (hasMorePages && (pagesReturned++ < 10))
         {
-            var postBody = issueAndPRQuery.ToJsonText();
+            var postBody = JsonConvert.SerializeObject(issueAndPRQuery);
+
+            var requestHeaders = new HttpHeaders
```

Comment:
This code will never work. `HttpHeaders` is an abstract class, and if you intend to instantiate `HttpRequestHeaders` there's no .ctor. I suggest, you instead leave the call as it was, and reformat the new lines as desired.